packages/url_launcher/url_launcher_web/README.md

```diff
@@ -4,11 +4,15 @@ The web implementation of [`url_launcher`][1].
 
 ## Usage
 
+### Import the package
```

Comment:
Since we're publishing to pub, we can just have people depend on it without using the `git` dependency

Reply:
Yep, I totally changed that in 0.1.0 :P